evap/staff/importers.py

```diff
@@ -505,13 +505,10 @@ def set_existing_course_to_merge_or_add_errors(self, semester, evaluation_data):
             if merge_hindrances:
                 self.errors[ImporterError.COURSE].append(
                     format_html(
-                        _(
-                            "Course {} ({}) does already exist in this semester, but the courses can not be merged for the following reasons:{}."
-                        ).format(
-                            evaluation_data.name_en,
-                            evaluation_data.name_de,
-                            "".join(f"<br /> - {msg}" for msg in merge_hindrances),
-                        )
+                        "Course {} ({}) does already exist in this semester, but the courses can not be merged for the following reasons:{}.",
```

Comment:
The translation here was lost in translation

Additionally, it would be nice if the German name would be first in the German translation, named format arguments should work for that